src/pages/InputDemo/InputDemo.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { TextField, SelectField, RadioField } from '../../components';
+import { selectOptions, radioOptionsCricket, radioOptionsFootball } from '../../configs/constants'
+
+class InputDemo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      sport: '',
+      cricket: '',
+      football: '',
+    };
+  }
+
+  handleNameChange = (e) => {
+    this.setState({ name: e.target.value })
+    console.log(this.state);
```

Comment:
remove unnecessary console from handlers and callbacks in InputDemo.jsx
Add single console in render method